src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const DELETE_BOOK = 'bookstore/books/DELETE_BOOK';
+
+const booksArray = [];
+
+const booksReducer = (state = booksArray, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.book,
+      ];
+    case DELETE_BOOK:
+      return state.filter((book) => book.id !== action.book.id);
```

Comment:
- [ ] [REQUIRED] Please, pay attention on your parameters, if you check your deleteBook Action Creators, it returns `{type: DELETE_BOOK, id}` so here, `state.filter((book) => book.id !== action.book.id)` should be `state.filter((book) => book.id !== action.id)`

Reply:
Thank you, I will fixed now